src/workspaces/ShareWorkspaceModal/Collaborator.test.ts

```diff
@@ -206,4 +207,81 @@ describe('a Collaborator component', () => {
     fireEvent.click(writterPermission);
     expect(setAcl).toHaveBeenCalledTimes(1);
   });
+
+  describe('only allows owners and project owners to share with additional permissions', () => {
+    const setAcl = jest.fn();
+    const aclMock = asMockedFn(setAcl);
+    const item: AccessEntry = {
+      email: 'user1@test.com',
+      pending: false,
+      canShare: true,
+      canCompute: true,
+      accessLevel: 'WRITER',
+    };
+    const acl = [item];
+
+    it('allows an owner to share with additional permissions', async () => {
```

Comment:
Nit. If a function doesn't `await` anything, it doesn't need to be `async`.

```suggestion
    it('allows an owner to share with additional permissions', () => {
```